CMakeLists.txt

```diff
@@ -1,23 +1,31 @@
 # Copyright 2011-2018 Sebastian A. Mueller
 cmake_minimum_required(VERSION 2.8)
-set (CMAKE_CXX_FLAGS "-std=c++1z -pthread -Wall -Wextra -pedantic -O1 -g -I")
```

Comment:
Here I used g++ optimization-flag '-O1' because it gave fastest execution of the tests on my machines.